megalista_dataflow/mappers/ads_user_list_pii_hashing_mapper.py

```diff
@@ -39,8 +39,19 @@ def __init__(self):
         self.logger = logging.getLogger("megalista.AdsUserListPIIHashingMapper")
 
     def _hash_user(self, user, hasher):
-
+        hashable_keys = ("email",
+                         "mailing_address_first_name",
+                         "mailing_address_last_name",
+                         "mailing_address_country",
+                         "mailing_address_zip",
+                         "phone",
+                         "mobile_device_id",
+                         "user_id")
         hashed = {}
+        # include non PII keys
+        for k,v in user.items():
+          if k not in hashable_keys:
```

Comment:
Not sure if the `not in` is intended. Shouldn't you add to the `hashed` dictionary only hashable keys?

Reply:
It is intended, the variable name is causing the confusion. I will refactor it.